src/app/pages/homepage/homepage.html

```diff
@@ -55,6 +55,23 @@ <h2>Optimized for Angular</h2>
       <p>Built by the Angular team to integrate seamlessly with Angular.</p>
     </div>
   </div>
+
+  <div style="width: 75%">
+  <app-carousel [itemWidth]="250" [itemHeight]="250">
+    <a carousel-item class='docs-carousel-item-container' *ngFor="let guide of guideItems.getAllItems()"
```

Comment:
can we call the class `docs-carousel-item`? `docs-carousel-item-container` makes it sound like a wrapper around this element